### PR TITLE
Fix #1015 - Survey background colors are now purple when user is premium

### LINKED ISSUE #1015: The “How can we improve Relay for you? Let us know.” link for premium accounts is white and almost invisible
#### Build:
- Stage;

#### Affected Platforms:
- Windows 10, Android 9;

#### Browsers tested on:
- Firefox, latest version;

#### Prerequisites:
- Be signed in into a Firefox Relay account with an active subscription;

#### Steps to reproduce:
1. Go to https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net/accounts/profile/ ;
2. If there is no survey banner present, go to the FAQ page;
3. Observe the survey banner at the top of the page;

#### Expected result:
- The banner should display the link for the survey;

#### Actual result:
- The banner and the link are displayed, but the link is written with white on white background and is therefore almost invisible;

#### Notes:
- The background color changes to white for premium users, but the color of the link remains white;
image
image

### PR DESCRIPTION
**Pull request by maxxcrawford**
Preview: 

image

### CODE REVIEW

File: static/scss/partials/banners.scss [hunk 1]
```diff
@@ -6,6 +6,10 @@
     ul {
         list-style-type: none;
     }
+    
+    .is-premium & {
```
Comment (codemist) [inline static/scss/partials/banners.scss:10]: What does the `&` at the end of this class do?

Comment (maxxcrawford) [reply static/scss/partials/banners.scss:10]: It's a [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector), which is pure SCSS Magic™. It's the same as writing `.is-premium .recruitment-banner`.

Comment (maxxcrawford) [reply static/scss/partials/banners.scss:10]: In this case, `.is-premium` is being added to the `<body>`, so it's influencing/cascading downstream to the banners.

Comment (codemist) [reply static/scss/partials/banners.scss:10]: Ahhh I see! Gotcha

**Review by codemist — APPROVED** [review]

**Pull request merged**